style.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: 'Inter';
+  src:
+    url('./fonts/Inter/Inter-Regular.woff') format('woff'),
+    url('./fonts/Inter/Inter-Regular.woff2') format('woff2'),
+    url('./fonts/Inter/Inter-SemiBold.woff2') format('woff2'),
+    url('./fonts/Inter/Inter-Bold.woff2') format('woff2'),
+    url('./fonts/Inter/Inter-ExtraBold.woff2') format('woff2');
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Inter, 'Times New Roman', Times, serif;
+  color: #344563;
+  background-color: #fff;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 20px;
+}
+
+header > * {
+  margin: 0 40px;
+  flex-shrink: 0;
+}
```

Comment:
- [ ] Nice to display as flex as suggested however, Instead of using the header to style your heading as well, I recommend you to include a class inside the header to use it for styling instead of styling the header.